libraries/helpers.rb

```diff
@@ -0,0 +1,21 @@
+module Resolver
+  module Cookbook
+    module Helpers
+      def resolver_resolv_conf_default
+        '/etc/resolv.conf'
```

Comment:
```suggestion
        '/etc/resolv.conf'.freeze
```

These strings should probably be frozen

Reply:
That's a good point that I'd not considered for this use before, where we don't need any logic these can become constants instead of methods.